sdk/appconfiguration/app-configuration/review/app-configuration.api.md

```diff
@@ -171,117 +107,87 @@ export interface GetKeyValueHeaders {
     syncToken?: string;
 }
 
-// @public
-export type GetKeyValueResponse = ConfigurationSetting & GetKeyValueHeaders & {
-    _response: coreHttp.HttpResponse & {
-        parsedHeaders: GetKeyValueHeaders;
-        bodyAsText: string;
-        parsedBody: ConfigurationSetting;
-    };
-};
-
 // @public
 export interface GetKeyValuesHeaders {
     syncToken?: string;
 }
 
-// @public
-export type GetKeyValuesResponse = KeyValueListResult & GetKeyValuesHeaders & {
-    _response: coreHttp.HttpResponse & {
-        parsedHeaders: GetKeyValuesHeaders;
-        bodyAsText: string;
-        parsedBody: KeyValueListResult;
-    };
-};
-
 // @public
 export interface GetRevisionsHeaders {
     syncToken?: string;
 }
 
 // @public
-export type GetRevisionsResponse = KeyValueListResult & GetRevisionsHeaders & {
-    _response: coreHttp.HttpResponse & {
-        parsedHeaders: GetRevisionsHeaders;
+export interface HttpConditionalFields {
+    ifMatch?: string;
+    ifNoneMatch?: string;
+}
+
+// @public (undocumented)
+export interface HttpResponseField<HeadersT> {
+    _response: HttpResponse & {
+        parsedHeaders: HeadersT;
         bodyAsText: string;
-        parsedBody: KeyValueListResult;
     };
-};
+}
 
 // @public
-export interface KeyValueListResult {
-    items?: ConfigurationSetting[];
-    nextLink?: string;
+export interface ListConfigurationSettingPage extends HttpResponseField<GetKeyValuesHeaders> {
+    // (undocumented)
```

Comment:
Pointing out missing docs since this one is a member field.

Reply:
Got it.